chef/cookbooks/keystone/recipes/server.rb

```diff
@@ -503,6 +503,10 @@
   openstack_command <<  " --os-identity-api-version #{node[:keystone][:api][:version]} --os-project-domain-id default --os-user-domain-id default"
 end
 
+if keystone_insecure
```

Comment:
Nitpicking: I wouldn't put an empty line before that block (as the previous block also has no empty line before it).